platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/event/tipset/TipsetEventCreatorTests.java

```diff
@@ -894,9 +919,11 @@ void noStaleEventsAtCreationTimeTest() {
         final EventCreator eventCreator =
                 buildEventCreator(random, time, addressBook, nodeA, () -> new ConsensusTransactionImpl[0]);
 
-        // FUTURE WORK: expand to cover birthRound for determining ancient.
-        eventCreator.setNonAncientEventWindow(
-                new NonAncientEventWindow(1, 100, 0 /* ignored in this context */, AncientMode.GENERATION_THRESHOLD));
+        eventCreator.setNonAncientEventWindow(new NonAncientEventWindow(
+                1,
+                100,
+                1 /* ignored in this context */,
```

Comment:
I see you changed the expired threshold to `1`, but the comment still says the value is ignored.

Out of curiosity, was this an unintentional change, or is this value maybe no longer ignored?

Reply:
The value is the minimum expiration threshold. There is a problem in that it is checked against `ConsensusConstants.FIRST_ROUND` when in ancient mode.  If the value is less, an `IllegalArgumentException` is thrown.  0 < 1.  Since the value is ignored for the unit test, setting it to 1 passes the comparison both to the minimum round and minimum generation (0).